nox/_option_set.py

```diff
@@ -157,6 +157,7 @@ def make_flag_pair(
     name: str,
     enable_flags: tuple[str, str] | tuple[str],
     disable_flags: tuple[str],
+    default: bool = False,
```

Comment:
```suggestion
```

The parameter can be passed with kwargs.

Reply:
I tried that first time round, but `make_flag_pair` passes the kwargs to both the enable and disable options so they cancel each other out. The default in the above case is only passed into the enable option.

Reply:
Oh, right.